resident/resident-service/src/test/resources/bootstrap.properties

```diff
@@ -27,7 +27,7 @@ mosip.notification.language-type=both
 mosip.notificationtype=SMS|EMAIL
 resident.notification.emails=girish.yarru@mindtree.com
 resident.notification.message=Notification has been sent to the provided contact detail(s)
-config.server.file.storage.uri=http://localhost:8081
+config.server.file.storage.uri=https://dev2.mosip.net/config/resident/mz/develop
```

Comment:
why is this change required for test bootstrap, we cannot assure that this api is always available

Reply:
The entire tests are like this. It expects such url to connect. Either we need to skip these tests and make it work for now.